ovos_color_parser/models.py

```
@@ -55,9 +55,16 @@ def hex_str(self) -> str:
-        g = int(hex_str[2:4], 16)
-        b = int(hex_str[4:6], 16)
+        if len(hex_str) == 6:
+            r = int(hex_str[0:2], 16)
+            g = int(hex_str[2:4], 16)
+            b = int(hex_str[4:6], 16)
+        elif len(hex_str) == 3:
+            r = int(hex_str[0:1] * 2, 16)
+            g = int(hex_str[1:2] * 2, 16)
+            b = int(hex_str[2:3] * 2, 16)
+        else:
+            raise ValueError(f"Invalid hex sting {hex_str}")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)